src/pages/projects/forms/ProjectDetailsForm.tsx

```diff
@@ -179,6 +186,31 @@ const ProjectDetailsForm: FC<Props> = ({ formik, project, isEdit }) => {
               ),
             }}
           />
+          <NetworkSelector
+            value={formik.values.default_project_network}
+            project={project ? project.name : "default"}
```

Comment:
```suggestion
            project={"default"}
```
We should always use the networks from the default project for project creation.